src/MainPage.jsx

```diff
@@ -309,13 +337,35 @@ class App extends Component {
       );
     } else {
       return (
-        <CommonWorkflows
-          onClickUpdateLineItems={this.updateLineItems}
-          onClickCollectCardPayments={this.collectCardPayment}
-          onClickSaveCardForFutureUse={this.saveCardForFutureUse}
-          onClickCancelPayment={this.cancelPendingPayment}
-          cancelablePayment={cancelablePayment}
-        />
+        <>
+          <CommonWorkflows
+            workFlowDisabled={this.isWorkflowDisabled()}
+            onClickCollectCardPayments={() =>
+              this.runWorkflow("collectPayment", this.collectCardPayment)
+            }
+            onClickSaveCardForFutureUse={() =>
+              this.runWorkflow("saveCard", this.saveCardForFutureUse)
+            }
+            onClickCancelPayment={this.cancelPendingPayment}
```

Comment:
Is there a reason that cancelling a pending payment is not a "workflow" in the same sense that collecting payment is?

Reply:
I think it's because this is a smaller action, and tied to the "collect payment" workflow.